WordPress/Classes/ViewRelated/Post/PostCardTableViewCell.m

```diff
@@ -84,6 +77,24 @@ - (UIView *)hitTest:(CGPoint)point withEvent:(UIEvent *)event
     return [super hitTest:point withEvent:event];
 }
 
+- (void)didMoveToSuperview
+{
+    [super didMoveToSuperview];
+    if (self.didPreserveStartingConstraintConstants) {
```

Comment:
One minor comment here @aerych!

There's a flow in which `[self superview]` might be *nil*. [Screenshot of the trace here]( https://cloudup.com/cBZW128Mnoh)

Since the view might not be, yet, attached to it's superview, i think it might cause a bad side effect in the `preserveStartingConstraintConstants` call right?